Makefile

```diff
@@ -157,17 +157,20 @@ check-clippy: install-clippy
 	cargo +$(RUSTV) check --all --all-targets --all-features --tests $(VERBOSE_FLAG)
 	cargo +$(RUSTV) clippy --all --all-targets --all-features --tests $(VERBOSE_FLAG) -- -D warnings -A clippy::upper_case_acronyms
 
+build_smartstreams:
+	rustup target add wasm32-unknown-unknown
+	pushd src/smartstream/examples; cargo build --release; popd || true
```

Comment:
Don't build everything.  Build specific package as done in previous version:
```
build_test:
	cargo build --release --target wasm32-unknown-unknown --package fluvio-filter-test
```

Reply:
I went ahead and added the `--package` flag, but two things to note:

- Since this task changes directories into `src/smartstream/examples` using pushd, it was already only building the filter, and
- I added a `src/smartstream/examples/.cargo/config.toml` that ensures the examples are always built with `wasm32-unknown-unknown`, which is fine because that examples directory is its own workspace